docs/sources/clients/promtail/stages/cri.md

```diff
@@ -46,4 +46,5 @@ The following key-value pairs would be created in the set of extracted data:
 
 - `content`: `message`
 - `stream`: `stdout`
-- `timestamp`: `2019-04-30T02:12:41.8443515`
+- `flags`: `xx`
```

Comment:
Do we output the flags field? It seem to be only used as an indicator if a partial or full line occurred so it can join the partials.

Reply:
@hterik see above

Reply:
It does get output in the labels. See https://github.com/grafana/loki/blob/main/clients/pkg/logentry/stages/extensions.go#L19 
I'm not sure if it is intended though or just part of the implementation leaking through? Should i remove this part?

I do believe that the area around how flags and multiline are treated is lacking a bit of documentation generally, but that's out of scope for this PR.